store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/AccumuloStore.java

```diff
@@ -137,6 +141,31 @@ public void initialise(final Schema schema, final StoreProperties properties)
         TableUtils.ensureTableExists(this);
     }
 
+    @Override
+    public void validateSchemas() {
+        super.validateSchemas();
+
+        final ValidationResult validationResult = new ValidationResult();
+
+        getSchemaElements().entrySet().forEach(entrySet -> entrySet.getValue().getProperties().forEach(propertyName -> {
+            final Serialiser serialisation = entrySet.getValue().getPropertyTypeDef(propertyName).getSerialiser();
+
+            if (serialisation != null && !(serialisation instanceof ToBytesSerialiser)) {
```

Comment:
The null check was a safety habit, but agreed the Null is checked and error is logged in the super.validateSchemas() and then thrown before this.